pandas/core/base.py

```diff
@@ -1572,5 +1573,9 @@ def duplicated(self, keep='first'):
     # ----------------------------------------------------------------------
     # abstracts
 
+    # Removed from type checking because there is currently no good way to
+    # handle multiple inheritance with incompatible signatures with mypy
+    # https://github.com/python/mypy/issues/2125
+    @no_type_check
```

Comment:
`pandas.core.series.Series` inherits from `base.IndexOpsMixin` and `generic.NDFrame`. The signature of this function conflicts with the definition in `pandas.core.generic.NDFrame`. The other signature includes `verify_is_copy=True` as a default argument. 

Rather than ignore this for type checking, it would also work to bring the two signatures into alignment, but I wasn't comfortable doing that in this context. 

It's probably better to have two mixins using the same method name share signatures, but ultimately that's a design/architecture choice. @jreback Thoughts?

Reply:
I think OK to align signatures, especially since this just raises anyway